docs/kit/adoption-view/page_adoption-view.md

```diff
@@ -36,13 +62,113 @@ The main difference between the EDC and the previous connectors of the [IDSA][id
+- [CX - 0044 ECLASS][CX044]
+- [CX - 0047 DCM Data Model Material Demand & Capacity Group][CX0047]
+- [CX - 0059 Triangle Behavioral Twin Endurance Predictor][CX0059]
+
+### Terminology
+
+![domain-model](images/domain-model.png)
+
+> The shown picture illustrates only a generic view of the Domain Model and is not intended to show all aspects of the project.
+
+#### Extensions
+
+There are different extenions for the Connector, e.g. for the Data Plane. This enables various tranfer modes like httpData or via blob-storage.
+
+#### Data Plane
+
+The Data Plane handles several forms of actual data exchange by utilizing various extensions.
+
+#### Control Plane
+
+The Control Plane handles meta data exchange with other components and Connectors, as well as transfer of access tokens
+
+#### Data Assets
+
+Data Sources (databases, files, cache information, etc.) are connected to the Connector and are represented by Data Assets. For each asset, a [`DataAddress`](#data-address) needs to be resolvable.
+
+#### Data address
+
+A data address is a pointer into the physical storage location where an asset will be stored.
+
+#### Policy Definition
+
+A standardized set of policies can be used to define actions regarding access to and usage of assets. These actions can be limited further by constraints (temporal or spatial) and duties ("e.g. deletion of the data after 30 days").
+
+#### Contract Definition
+
+By combining Assets and Policies, Contracts for data offerings are defined. These Contracts need to be accepted by consuming participants (Connectors) for the data exchange to take place.
+
+#### Contract offer
+
+The contract offer is a dynamic representation of the [`ContractDefinition`](#contract-definition)
+for a specific consumer and serves as protocol's data transfer object (DTO) for a particular contract negotiation.
+Contract offers are not persisted and will be regenerated on every request. The connector acting as data provider will
+generate contract offers only for contract definitions dedicated to the organization or data space participant
+operating the requesting connector acting as data consumer. A contract offer is always related to a single asset of
+the `ContractDefinition` object (e.g. for a `ContractDefinition` containing three `Asset` objects, the connector will
+generate three `ContractOffer` objects).
+
+#### Contract negotiation
+
+A `ContractNegotiation` captures the current state of the negotiation of a contract (`ContractOffer` ->
+`ContractAgreement`) between two parties. This process is inherently asynchronous.
+
+#### Contract agreement
+
+A contract agreement represents the agreed-upon terms of access and usage of an asset's data between two data space
+participants, including a start and an end date and further relevant information.
+
+#### Transfer process
+
+After a successful contract negotiation, a `DataRequest` is sent from a consumer connector to a provider connector to
```

Comment:
This could be worded a bit differently. Afaik there's no automatic mechanism that created DataRequests after a successful ContractNegotiation. Maybe something like this would be more accurate?
`After a successful contract negotiation, DataRequests can be sent from the consumer...`